torchgeo/datamodules/sustainbench_crop_yield_prediction.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Sustainbench Crop Yield Prediction datamodule."""
+
+from typing import Any
+
+from ..datasets import SustainBenchCropYieldPrediction
+from .geo import NonGeoDataModule
+
+
+class SustainbenchCropYieldDataModule(NonGeoDataModule):
+    """LightningDataModule for Sustainbench Crop Yield Prediction dataset."""
+
+    def __init__(
+        self, batch_size: int = 32, num_workers: int = 0, **kwargs: Any
+    ) -> None:
+        """Initialize a new SustainbenchCropYieldDataModule instance.
+
+        Args:
+            batch_size: Size of each mini-batch.
+            num_workers: Number of workers for parallel data loading.
+            **kwargs: Additional keyword arguments passed to
+                :class:`~torchgeo.datasets.SustainBenchCropYieldPrediction`.
+
+        .. versionadded:: 0.5
```

Comment:
This should be in the class docstring. It's not just `__init__` that was added, it's the entire class